src/main/java/seedu/address/model/volunteer/Gender.java

```diff
@@ -0,0 +1,60 @@
+package seedu.address.model.volunteer;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Volunteer's gender in the address book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidGender(String)}
+ */
+public class Gender {
+
+    public static final String MESSAGE_GENDER_CONSTRAINTS =
+            "Gender should only contain 'male' or 'female', and it should not be blank";
+
+    /*
+     * Gender must only be male or female
+     */
+    public static final String GENDER_MALE_VALIDATION_REGEX = "male";
```

Comment:
Do we want users to enter the whole word? 'Male and Female'. Or should we allow them to type 'M' and 'F' for convenience. Then when storing and retrieving, we can always do a if-else to convert the char to the whole word.